charts/seed-monitoring/charts/grafana/dashboards/owners/node-pool-dashboard.json

```diff
@@ -46,7 +46,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "avg(sum(rate(node_cpu_seconds_total{mode!=\"idle\"}[$__rate_interval])) by (node) + on (node) group_right sum(kube_node_labels{label_worker_gardener_cloud_pool=~\"$worker_group\"}) by (label_worker_gardener_cloud_pool, node) - 1) by (label_worker_gardener_cloud_pool)",
+          "expr": "avg by (label_worker_gardener_cloud_pool) (\n    sum by (node) (rate(node_cpu_seconds_total{mode!~\"idle|iowait|steal\"}[$__rate_interval]))\n  + on (node) group_right ()\n    sum by (label_worker_gardener_cloud_pool, node) (\n      0 * kube_node_labels{label_worker_gardener_cloud_pool=~\"$worker_group\"}\n    )\n)",
```

Comment:
There are also other occurrences of

```
node_cpu_seconds_total{mode!~\"idle\"}
```

https://github.com/gardener/gardener/blob/d31e21c7254169e267ad96655e64f6cdd76dc095/charts/seed-monitoring/charts/grafana/dashboards/owners/node-details-dashboard.json#L265-L273

Don't they need to be adapted to make sure that all dashboards show the same CPU usage value? Otherwise I expect some minor differences.